src/tutorials/nft_cosmetics.md

```diff
@@ -0,0 +1,380 @@
+---
+id: nft_cosmetics
+name: NFT Cosmetics
+title: NFT Cosmetics
+tags:
+    - Tutorial
+---
+<!-- vale Google.Acronyms = NO -->
+# NFT Cosmetics
+
+<div class="mt-video" style="width:100%">
+    <video autoplay muted playsinline controls loop class="center" style="width:100%">
+        <source src="/img/NFTCosmetics/preview.mp4" type="video/mp4" />
+    </video>
+</div>
+
+## Overview
+
+In this tutorial, you will learn how to use NFTs (Non-fungible Tokens) as cosmetics. You will create a system that allows players to walk into a trigger and get a random cosmetic attached to them. You will be using an NFT cosmetics collection that has already been created, however, creating your own can also be done and used for this tutorial.
+
+- **Skills you will learn:**
+    - Reading NFTs using the Blockchain API.
+    - Attaching objects to a socket on a player.
+    - Changing the colors of a static mesh.
+    - Creating a 3D progress bar.
+
+## Using an Existing NFT Collection
+
+This tutorial will use an existing NFT collection that uses the cardboard boxes from **Core Content**. For example, looking at NFT #3 in the collection, we can see it has attributes for **Box** and **Color**. The **Box** attribute is the row index from the data table of templates. This could be your data table of templates that make up how the cosmetic looks.
```

Comment:
Adding a `#` creates  a link to a pull request. May need to use backslash before it to cancel that or something